conda/activate.py

```diff
@@ -84,9 +106,18 @@ def __init__(self, shell, arguments=None):
             self.command_join = '\n'
 
             self.unset_var_tmpl = 'del $%s'
+            self.unexport_var_tmpl = self.unset_var_tmpl
             self.export_var_tmpl = "$%s = '%s'"
+            self.set_var_tmpl = self.export_var_tmpl
             self.run_script_tmpl = 'source "%s"'
 
+            self.post_tmpl = dedent("""\
+                post_pipeline = !(@(_CONDA_EXE) shell.xonsh %spost)
+                post_stdout = _raise_pipeline_error(post_pipeline)
```

Comment:
`_raise_pipeline_error` is a function defined in https://github.com/conda/conda/blob/4.5.0rc0/conda/shell/conda.xsh#L24-L31. We shouldn't assume it's always available.

Reply:
Well I would also expect any changes to the individual shell scripts to also make mirrored updates to the applicable `post_tmpl`. This is why I consider this implementation to be relatively dumb.